src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -193,6 +207,13 @@ public Command driveJoystick(
 				});
 	}
 
+	public void simpleDrive(double forward, double strafe, Rotation2d rotation) {
```

Comment:
Where is this method used?

Reply:
In drivecommand. This method will be deleted once drivecommand is switched to a better drive method